code/modules/power/apc.dm

```diff
@@ -750,7 +750,7 @@
 
 /obj/machinery/power/apc/ui_act(action, list/params)
 	. = ..()
-	if(. || !can_use(usr, TRUE) || locked)
+	if(. || !can_use(usr, TRUE) || locked & (!usr.has_unlimited_silicon_privilege))
```

Comment:
```suggestion
	if(. || !can_use(usr, TRUE) || (locked && !usr.has_unlimited_silicon_privilege))
```

Reply:
Why is this better ?

Reply:
& is a binary operation, it's boolean math.
&& is logical operation

& evaluates both sides of the operation.
&& evaluates the left side of the operation, if it's true, it continues and evaluates the right side.